api overview draft (#2771)
wix-incubator/wix-code-docs: guides/Overview.md

# Introduction

> Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site’s functionality. Use Velo APIs to fully customize your site, from frontend design to backend functionality.

Reviewer: ```suggestion
Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site, from frontend design to backend functionality.
```
This is a strong intro but it felt to me like the last two sentences partially repeat the same point. This is one way to combine them but you could also choose another, like "Velo empowers you to fully customize your site, from frontend design to backend functionality."

> The API reference provides detailed documentation and code examples for all APIs, functions and event handlers, and properties that you can use to interact with various aspects of your site. To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. 

Reviewer: ```suggestion
The API reference provides detailed documentation and code examples for all functions, event handlers, and properties that you can use to interact with various aspects of your site. To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. 
```
The commas and "and"s in the first sentence were a little confusing. Maybe this edit resolves the issue. (In any event I don't think APIs are something distinct from functions and event handlers here).

With Velo you can:

> + **Access Site Elements**: Use the [Wix Editor Elements($w)](https://www.wix.com/velo/reference/$w) API to work with your site's elements and handle user interactions.

Reviewer: ```suggestion
+ **Access site elements**: Use the [Wix Editor Elements($w) API](https://www.wix.com/velo/reference/$w) to work with your site's elements and handle site visitor interactions.
```

I moved "API" into the link text for consistency with the links below.

> + **Manage Database Collections**: Programmatically interact with your site's database collections using the [Data API](https://www.wix.com/velo/reference/wix-data).

Reviewer: ```suggestion
+ **Manage database collections**: Programmatically interact with your site's database collections using the [Wix Data API](https://www.wix.com/velo/reference/wix-data).
```

> + **Create Custom eCommerce Solutions**: The [Wix eCommerce API](https://www.wix.com/velo/reference/wix-ecom-backend) allows you to manage your products, orders, fulfillment, and more.

Reviewer: ```suggestion
+ **Create custom eCommerce solutions**: The [Wix eCommerce API](https://www.wix.com/velo/reference/wix-ecom-backend) allows you to manage customers' carts, checkouts, orders, and more.
```

Products is part of Stores, not eCom, so I suggested a change to the list.

>**New to Velo?** Check out our [getting started guide](https://dev.wix.com/docs/develop-websites/articles/getting-started/getting-started-with-code) to get you quickly up and running with code.

## Release Notes

> 

Reviewer: Consider changing this sentence so "release notes" isn't capitalized.:

"Check out our [Release Notes](https://www.wix.com/velo/reference/release-notes/release-notes) to see a list of all the latest changes, updates, and additions to our API Reference."
